### PR TITLE
feat: api and docs for processors

### PR DESCRIPTION
**Pull request by bennypowers**
support for scss, postcss, etc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,8 +47,11 @@
     "lit": "^2.0.0-rc.2",
     "lit-element": "^2.0.0",
     "memfs": "^3.2.2",
+    "node-sass": "^6.0.1",
```
Comment (jpzwarte) [inline package.json:50]: I believe `node-sass` is deprecated. `sass` is the supported way.

File: packages/lit-css-loader/README.md [hunk 2]
```diff
@@ -23,19 +23,17 @@ In the mean time, enjoy importing your CSS into your component files.
...
 ## Options
 
-|Name|Accepts|Default|
-|-----|-----|-----|
-|`include`|Array of glob of files to include.|`['**/*.css']`|
-|`exclude`|Array of glob of files to exclude. |`undefined`|
-|`uglify`|Boolean or Object of [uglifycss](https://www.npmjs.com/package/uglifycss#api) options.|`false`|
-|`specifier`|Package to import `css` from|`lit`|
-|`tag`|Name of the template-tag function|`css`|
+| Name        | Accepts                                                                                | Default        |
```
Comment (jpzwarte) [inline packages/lit-css-loader/README.md:26]: `transform` is not an option for the loader?

File: packages/rollup-plugin-lit-css/README.md [hunk 3]
```diff
@@ -22,13 +23,13 @@ In the mean time, enjoy importing your CSS into your component files.
...
 ## Options
 
-|Name|Accepts|Default|
-|-----|-----|-----|
-|`include`|Array of glob of files to include.|`['**/*.css']`|
-|`exclude`|Array of glob of files to exclude. |`undefined`|
-|`uglify`|Boolean or Object of [uglifycss](https://www.npmjs.com/package/uglifycss#api) options.|`false`|
-|`specifier`|Package to import `css` from|`lit`|
-|`tag`|Name of the template-tag function|`css`|
+| Name        | Accepts                                                                                | Default        |
```
Comment (jpzwarte) [inline packages/rollup-plugin-lit-css/README.md:26]: Shouldn't `transform` be documented here as well?

**Review by jpzwarte — APPROVED** [review]

File: packages/rollup-plugin-lit-css/README.md (hunk 3 above)
Comment (bennypowers) [reply packages/rollup-plugin-lit-css/README.md:26]: Thanks!

File: packages/esbuild-plugin-lit-css/README.md [hunk 4]
```diff
@@ -96,14 +96,14 @@ class CSSinCSS extends FASTElement {}
 ### Usage with Sass, Less, PostCSS, etc.
 
 To load scss files:
-1. Specify the [`js` loader in your esbuild config](https://esbuild.github.io/api/#loader) for `.scss` files,
-2. Specify the `filter` option to `litCssPlugin` to include scss files
-2. Define a `transform` function in the plugin options.
+1\. Specify the [`js` loader in your esbuild config](https://esbuild.github.io/api/#loader) for `.scss` files,
+2\. Specify the `filter` option to `litCssPlugin` to include scss files
+2\. Define a `transform` function in the plugin options.
```
Comment (jpzwarte) [inline packages/esbuild-plugin-lit-css/README.md:101]: 3? :)

Comment (bennypowers) [reply packages/esbuild-plugin-lit-css/README.md:101]: yes, and markdown automatically fixes that for you, so whatev

**Pull request merged**